packages/nodes/table/src/components/TableCellElement/TableCellElementResizable.tsx

```diff
@@ -82,51 +99,69 @@ export const useTableCellElementResizableProps = ({
     setIsResizingRight(true);
   };
 
-  const onResize: ResizableProps['onResize'] = (e, direction, ref) => {
+  const onResizeStart: ResizableProps['onResizeStart'] = (e, direction) => {
     if (direction === 'right') {
       setHoveredColIndex(colIndex);
+      setInitialColSize(colSizes[colIndex]);
+      setInitialSiblingColSize(colSizes[colIndex + 1] ?? null);
+    }
+  };
 
-      setResizingCol({
-        index: colIndex,
-        width: roundToStep(ref.offsetWidth, stepX),
-      });
+  const onResize: ResizableProps['onResize'] = (e, direction, ref) => {
+    if (direction === 'right') {
+      const newSize = roundToStep(ref.offsetWidth, stepX);
+
+      overrideColSize(colIndex, newSize);
+
+      if (initialColSize && initialSiblingColSize) {
+        overrideColSize(
+          colIndex + 1,
+          initialColSize + initialSiblingColSize - newSize
+        );
+      }
     } else {
-      setOverrideRowSize(roundToStep(ref.offsetHeight, stepY));
+      overrideRowSize(rowIndex, roundToStep(ref.offsetHeight, stepY));
     }
   };
 
-  const onResizeStop: ResizableProps['onResizeStop'] = (e, direction, ref) => {
+  const onResizeStop: ResizableProps['onResizeStop'] = (e, direction) => {
     if (direction === 'right') {
-      setTableColSize(
-        editor,
-        { colIndex, width: roundToStep(ref.offsetWidth, stepX) },
-        { at: findNodePath(editor, element)! }
-      );
+      colSizeOverrides.forEach((size, index) => {
+        setTableColSize(editor, { colIndex: index, width: size }, { at: path });
 
-      // Prevent flickering
-      setTimeout(() => setResizingCol(null), 0);
+        // Prevent flickering
+        setTimeout(() => overrideColSize(index, null), 0);
+      });
 
       setHoveredColIndex(null);
       setIsResizingRight(false);
     } else {
-      setTableRowSize(
-        editor,
-        { height: roundToStep(ref.offsetHeight, stepY) },
-        { at: findNodePath(editor, element)! }
-      );
-
-      // Prevent flickering
-      setTimeout(() => setOverrideRowSize(null), 0);
+      rowSizeOverrides.forEach((size, index) => {
+        setTableRowSize(
+          editor,
+          { rowIndex: index, height: size },
+          { at: path }
+        );
+
+        // Prevent flickering
+        setTimeout(() => overrideRowSize(index, null), 0);
+      });
     }
   };
 
+  const maxWidth =
```

Comment:
can be unit tested (extract function)

Reply:
Same question as https://github.com/udecode/plate/pull/2276/files/d399e0def7f32aa1dce45c59242e7523f8ac3e1f#r1135475137

Reply:
Let's forget this one